specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/invoice.json

```diff
@@ -80,860 +93,1224 @@
     },
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/downloadDocuments": {
```

Comment:
Downloading Invoice api, what do they take?

I believe in past we used to take url etc.

other than InvoiceId, nothing else should be present in the request body.

Also, do we have a singular invoice download link?